clicache/src/DataInput.cpp

```diff
@@ -878,8 +877,6 @@ namespace Apache
 
       void DataInput::Cleanup()
       {
```

Comment:
If this method is empty perhaps we can remove it?

Reply:
Done